### DISCUSSION TITLE
can this be used with selfhosted media server which has sonarr,radarr and automate downloading from there?

### DISCUSSION THREAD

**Discussion by Greatz08** [VueTorrent/VueTorrent#1543] [Q&A] [answered]
they support limited number of downloaders so i asked this question :-)

**Comment by Larsluph** [comment]
From what I'm aware, qBittorrent is supported with the Servarr environment.

Please note that VueTorrent and qBittorrent are 2 distinct projects and thus isn't related to VueTorrent in any way.

**Reply by Greatz08** [reply]
@Larsluph yea ik that both are different. Maybe you can ask arr community to add support for this awesome project, so that more will use this project instead of given options. Tbh i dont like max interface and option so i was using transmission which has simple to use interface but if this project can be intergrated with servarr environment then many will adopt it :-)) because many do torrenting and want automation so that from for example prowlarr we can directly initiate download in vuetorrent :)

**Reply by Larsluph** [reply, marked as answer]
I'm not sure we've understood each other. All of the Servarr apps use qBittorrent's API to manage torrent downloads. This API doesn't change depending on the WebUI you use.

VueTorrent is only a WebUI, meant to be used by humans to interface with that same API.

Think of VueTorrent as a "visual skin" over qbittorent. Every torrent-related features and management is done by qBittorrent, not VueTorrent.

**Reply by Greatz08** [reply]
@Larsluph now i get it , so its just visual skin of qbit torrent sorry my bad it was mentioned i didnt read carefully :-(( anyways as you mentioned if it uses same API as qbit torrent so will we be able to automate with all servarr solutions like homarr,radarr etc ? i am asking because you said it use same API as qbit torrent so if we provide vue torrent server link i think it should be able to automate download right? If yes then please put this information in BOLD TEXT so that many who have doubt about it in anyway will be able to understand it easily and will try out instantly :-))
